app/src/main/java/bunkou/tms/countries/Country.kt

```diff
@@ -0,0 +1,50 @@
+package bunkou.tms.countries
+
+import androidx.lifecycle.MutableLiveData
+import kotlinx.coroutines.delay
+import kotlin.random.Random
+
+
+interface Country {
+
+
+    var Olives: MutableLiveData<Int>
```

Comment:
If you have 6 fruits here these fruits would be in  all countries.If you want have different fruits in each country you should move variables in classes, remaining reapCrop method in interface and implement this method in your classes

Reply:
Thanks!